.travis.yml

```diff
@@ -12,25 +8,26 @@ env:
   - secure: z6WQVmKFTWf7in9J8764ic6gs5QyLctGYIDzstjSBXYrLTUfJjUwB7OAup1fAFXYUxtzkn6vjn3CIuy+5Nq7jL25Bwen4SJCPZtk9w/8acXUNDNlIFIak7T/jelOO11gHxHI6j2H+wj1BGxz1XZEkx8eeUN6389yvWgHQH01sbrQ7gXRgfmK+1pvUK9Li+4Mn9gBbOoqDxz2w8Wuazkk/FcmlYRYet4z8/y4FHwyc+Sa69utVJQAhDwJzngtb5WRdlWlrcFfyWhQ0eZ/shKxBcjekU5ZeLCh0av4ttB2IXVvLHzsr6vuCssRJqC7LHqiWEpXacWSf0ZOpMlAzwT+fPH5xXSl1kdc8vkpz8gYC5YDDMxkTrUoRv8vCYi/KWmYXNzdiAQn53aX741fcj5FV8v0ybmdPPQLfeQPDhCeHLlroWZsI5cgygxotIfeQzOY3MXSi+YqtZjmuHjAC1aINn9sLwYtCbV1ZVn3TxPaMyWjPlBuOaYoU6GkC+HxAcUy9QMVxBBzbVbOnaPri8NTXxHE58cIz+6GZVou4UDpEjjpi3Z8hieXu5cR0KKFVPofIQlgLaFZ9YgtAuXW6RP2k7ONYyCbW7bz39nnY6outFe96yR/HB4UKqBwc8dIUPfgzyoEyE1obnuNcsNf6UZf+B5ALM03ejQCx9KJuFPrZSw=
 jobs:
   include:
-  - stage: tests
-    before_install:
-    - pip install poetry
-    install:
-    - echo "Installing through poetry then running tests"
-    - poetry install
-    - pip install coveralls
-    script:
-    - flake8 dcicutils
-    - travis_wait coverage run --source=dcicutils -m pytest -vv test
-    - coveralls
-    deploy:
-      provider: script
-      script: poetry publish --build --username=$PYPI_USER --password=$PYPI_PASSWORD
-      on:
-        tags: true
-    after_success:
-    - echo "Finished running tests"
-    - |
+    - python: "3.6"
+    - python: "3.7"
+before_install:
```

Comment:
This is a different indentation level. Is that right? I don't know the yaml rules for this, but it doesn't look right.